### PR TITLE
Abolishes GBP + Code of Conduct tweaks

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do

Abolishes GBP and tweaks the code of conduct, largely to update it and add a new clause

## Why It's Good For The Repo

We dont really enforce GBP here. Some people have hundreds of points (me included), and it can be discouraging to newer contribs if they instantly have to go out of their way and fix things.

That said, this is not opening the floodgates to a spam of feature and balance PRs. Headcoders may close your PRs at will if your PR history is lackluster in regards to improving repo maintainability, but only in major circumstances such as only balance and feature PRs.

This change will not affect 99% of people.

### CODE REVIEW

**Review by hal9000PR — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -14,21 +14,23 @@ Under no circumstances are users to be attacked for their ideas or contributions
...
 
 ### PR Expectations
 
 All Pull Requests are expected to be tested prior to submission. If a submitted Pull Request fails to pass CI checks, the likelihood of it being merged will be significantly lower. If you can't take the time to compile/test your Pull Request, do not expect a warm reception.
 
-It is expected that contributors discuss larger changes on the forums prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Maintainers are not responsible for contributors wasting their time creating features nobody asked for.
+It is expected that contributors discuss larger changes on the forums prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Repo staff are not responsible for contributors wasting their time creating features nobody asked for.
```
Comment (Sirryan2002) [inline CODE_OF_CONDUCT.md:33]: Could you extend the platforms that changes should be discussed on in `discuss larger changes` to also include the repository discussions tab or would you prefer a forums thread be a *required* part of larger changes (perhaps due to the accessibility to the discussion by non devs?)

**Comment by S34NW** [comment]
I've been robbed

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (AffectedArc07) [reply CODE_OF_CONDUCT.md:33]: Can do

**Comment by matttheficus** [comment]
Single-handedly destroying the GBP economy.

**Comment by AffectedArc07** [comment]
> Single-handedly destroying the GBP economy.

image

**Review by Fox-McCloud — APPROVED** [review]

**Pull request merged**